frontend/style.css

```diff
@@ -364,13 +364,6 @@ input[type="checkbox"], input[type="radio"] {
 
 /** Zones **/
 
-#img,
-.card {
```

Comment:
You're still using this class (I think unnecessarily) in your code.

Have your searched the whole code base for uses of this? It was kinda everywhere in the past (though I've been pruning it out!)